extide/gradle/src/org/netbeans/modules/gradle/execute/GradleDaemonExecutor.java

```diff
@@ -383,6 +411,88 @@ private static String gradleExecutable() {
         return Utilities.isWindows() ? "bin\\gradle.bat" : "bin/gradle"; //NOI18N
     }
 
+    private static String[] unescapeParameters(String s) {
+        final int NULL = 0x0;
```

Comment:
I was trying to find a difference between this and `BaseUtilities.parseParameters` -- if it is there, please put a comment in the source, so maintainer(s) are aware about the important differences.

Reply:
Ah, thanks! I had a private copy elsewhere and forgot that it;s in BaseUtilities. I'll call it from there...

Reply:
Done.